src/Stratis.Bitcoin.Features.PoA/BasePoAFeatureConsensusRules/PoAHeaderSignatureRule.cs

```diff
@@ -100,15 +96,15 @@ public override async Task RunAsync(RuleContext context)
 
                         IEnumerable<PubKey> modifiedFederation = this.votingManager?.GetModifiedFederation(context.ValidationContext.ChainedHeaderToValidate).Select(m => m.PubKey) ?? genesisFederation;
 
-                        this.Logger.LogDebug($"Block {context.ValidationContext.ChainedHeaderToValidate}:{context.ValidationContext.ChainedHeaderToValidate.Header.Time} is signed by '{pubKeyForSig.ToHex()}' but expected '{pubKey}' from: { string.Join(" ", modifiedFederation.Select(pk => pk.ToHex()))}.");
+                        this.Logger.LogWarning($"Block {context.ValidationContext.ChainedHeaderToValidate}:{context.ValidationContext.ChainedHeaderToValidate.Header.Time} is signed by '{pubKeyForSig.ToHex()}' but expected '{pubKey}' from: { string.Join(" ", modifiedFederation.Select(pk => pk.ToHex()))}.");
 
                         break;
                     };
                 }
                 catch (Exception) { }
 
-                this.Logger.LogDebug("(-)[INVALID_SIGNATURE]");
-                PoAConsensusErrors.InvalidHeaderSignature.Throw();
+                //this.Logger.LogDebug("(-)[INVALID_SIGNATURE]");
+                //PoAConsensusErrors.InvalidHeaderSignature.Throw();
```

Comment:
hmm.. Did not expect this getting disabled. In the interest of block chain integrity it's probably not a good idea.

Reply:
Ah sorry you are right, cause we set the InsuffcientInformation we wont ban.